application/common/config/test.php

```diff
@@ -52,9 +52,9 @@
                 'host' => null,
             ],
         ],
-        'personnel' => [
-            'class' => 'tests\mock\personnel\Component',
-        ],
+//        'personnel' => [
```

Comment:
Particular reason these lines were commented out?

Reply:
Yes, to revert the default of using the Broker container rather than the mocks. I chose to leave those lines so that it's easy to use the mocks if needed. In reality, I could have completely removed the Personnel mocks, and might still do that.